docs/foundation/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Ethereum Foundation
+meta:
+  - name: description
+    content: Learn about the Ethereum Foundation (EF), a non-profit organization dedicated to supporting Ethereum and related technologies.
+  - property: og:title
+    content: Ethereum Foundation | Ethereum.org
+  - property: og:description
+    content: Learn about the Ethereum Foundation (EF), a non-profit organization dedicated to supporting Ethereum and related technologies.
+sidebar: false
+lang: en-US
+---
+
+# About the Ethereum Foundation
+
+<div>
+  <img :src="$withBase('/ef-logo.png')" class="ef-logo" />
+</div>
+
+The [Ethereum Foundation](http://ethereum.foundation/) (EF) is a non-profit organization dedicated to supporting [Ethereum](/what-is-ethereum/) and related technologies.
+
+The EF is not a company, or even a traditional non-profit. Their role is not to control or lead Ethereum, nor are they the only organization that funds critical development of Ethereum-related technologies. The EF is one part of a much larger [ecosystem](/community/).
+
+Learn more at [ethereum.foundation](http://ethereum.foundation/) or check out the [Ethereum Foundation Blog](https://blog.ethereum.org/) for the EF's latest news and announcements.
```

Comment:
```suggestion
```

Let's move this bit to the very bottom of the page, in order to suggest to the reader that there are more EF initiatives than just the 2 listed here